spinalcordtoolbox/reports/qc2.py

```diff
@@ -545,6 +546,11 @@ def sct_analyze_lesion(
                 slice_sc = im_sc_data[sagittal_slice]
                 slice_lesion = im_label_data_cur[sagittal_slice]
 
+                # Convert the sagittal slice to the original orientation
+                # 'x' because R-L direction (first in RPI --> 'x')
+                sagittal_slice = rpi_slice_to_orig_orientation(im_lesion.dim[0:3], orig_orientation,
```

Comment:
We could probably just pass `im_lesion.dim` and there would be no difference (for the suggested refactored code), since we're accessing it using 0/1/2 `axis` values.